ynr/apps/moderation_queue/tests/test_changes_for_review.py

```diff
@@ -257,6 +259,127 @@ def test_users_with_past_approved_edits_dont_need_review(self):
             PREVIOUSLY_APPROVED_COUNT + 1,
         )
 
+    def update_person(self, user, example_person):
+        response = self.app.get(
+            "/person/{person_id}/update".format(person_id=example_person.id),
+            user=self.user,
+        )
+
+        form_data = response.forms["person-details"]
+        form_data["favourite_biscuit"] = "Jammy Dodgers"
+        form_data["source"] = "A biscuit update for testing purposes"
+        form_data.submit()
+        example_person.refresh_from_db()
+        self.assertEqual(example_person.favourite_biscuit, "Jammy Dodgers")
+
+        form_data = response.forms["person-details"]
+        form_data["favourite_biscuit"] = "Chocolate Chip"
+        form_data["source"] = "Another biscuit update for testing purposes"
+        form_data.submit()
+        example_person.refresh_from_db()
+
+        self.assertEqual(example_person.favourite_biscuit, "Chocolate Chip")
+
+        form_data = response.forms["person-details"]
+        form_data["favourite_biscuit"] = "Coconut"
+        form_data["source"] = "Another biscuit update for good measure"
+        form_data.submit()
+        example_person.refresh_from_db()
+
+        self.assertEqual(example_person.favourite_biscuit, "Coconut")
+
+        versions_data = example_person.versions
+        self.assertEqual(len(versions_data), 3)
+        self.assertEqual(LoggedAction.objects.count(), 3)
+        self.assertEqual(
+            LoggedAction.objects.filter(action_type="person-update").count(), 3
+        )
+
+        response = self.app.get("/", user=self.user)
+        self.user.is_active = False
+        self.user.save()
+        response = self.app.get("/")
+        self.assertIn("Sign in to edit", response.text)
+
+    def revert_edit_to_person(self, example_person):
+        self.user = User.objects.get(id=3)
```

Comment:
I've tried getting a User which occasionally passes tests. If I create a User, I think there is an issue with logging in/authenticating the User despite `is_authenticate` and `is_active` = `True`

Reply:
The response for creating a user is empty whereas the response for getting an existing user returns the expected response, however I think that `get` relies on the local db.